docs/src/components/ComponentDoc/ComponentExample/ComponentExampleKnobs.tsx

```diff
@@ -23,6 +23,7 @@ const ComponentExampleKnobs = createComponent({
           {({ classes }) => (
             <CodeSnippet
               className={classes}
+              copyable={false}
```

Comment:
With #1477 was introduced an ability to copy contents of snippet, but there are no sence to copy values from knobs